test/acceptance/features/bindAppToService.feature

```diff
@@ -266,36 +266,36 @@ Feature: Bind an application to a service
             apiVersion: stable.example.com/v1
             kind: Backend
             metadata:
-                name: service-missing-app
+                name: $scenario_id-backend
                 annotations:
                     service.binding: path={.status.data.dbCredentials},objectType=Secret,elementType=map
             status:
                 data:
-                    dbCredentials: backend-secret
+                    dbCredentials: $scenario_id-secret
             """
         When Service Binding is applied
             """
             apiVersion: binding.operators.coreos.com/v1alpha1
             kind: ServiceBinding
             metadata:
-                name: service-binding-missing-app
+                name: $scenario_id-binding
             spec:
                 bindAsFiles: false
                 application:
-                    name: gen-missing-app
+                    name: $scenario_id
                     group: apps
                     version: v1
                     resource: deployments
                 services:
                 -   group: stable.example.com
                     version: v1
                     kind: Backend
-                    name: service-missing-app
+                    name: $scenario_id-backend
             """
-        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "service-binding-missing-app" should be changed to "True"
-        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "service-binding-missing-app" should be changed to "False"
-        And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding "service-binding-missing-app" should be changed to "ApplicationNotFound"
-        And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "service-binding-missing-app" should be changed to "False"
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "$scenario_id-binding" should be changed to "True"
```

Comment:
there is just a single service binding in this scenario - hence we can omit the binding name here.

Reply:
Fixed.